asp.tf

```diff
@@ -18,4 +18,16 @@ module "asp" {
   asp_name            = "${var.product}"
   ase_name            = "${local.ase_name}"
   tag_list            = "${local.tags}"
+}
+
+module "asp" {
```

Comment:
name needs to be different, divorce just added a `-2` suffix, but up to you if you can come up with a better name

Reply:
```suggestion
module "asp-rd" {
```

Reply:
Oops, yes of course. I forgot about modules needing unique names. Thanks